### PR TITLE
[15.0][FIX] sale_line_refund_to_invoice_qty, fix default value on aml

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

File: sale_line_refund_to_invoice_qty/models/account.py [hunk 1]
```diff
@@ -12,15 +12,17 @@ def _reverse_move_vals(self, default_values, cancel=True):
...
+        for vals in move_vals["line_ids"]:
+            vals[2].update({"sale_qty_to_reinvoice": sale_qty_to_reinvoice})
         return move_vals
 
 
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     sale_qty_to_reinvoice = fields.Boolean(
+        string="Sale qty to reinvoice",
```
Comment (RodrigoBM) [inline sale_line_refund_to_invoice_qty/models/account.py]: this is not necessary.
```suggestion
```
https://github.com/OCA/OCB/blob/16.0/odoo/fields.py#L507

**Review by RodrigoBM — CHANGES_REQUESTED** [review]

File: sale_line_refund_to_invoice_qty/models/account.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply sale_line_refund_to_invoice_qty/models/account.py]: Done!

**Review by RodrigoBM — CHANGES_REQUESTED** [review]
Hi @ChrisOForgeFlow ,

If you launch a wizard credit note and uncheck "This credit note will be reinvoiced" and  check credit method "Full refund and new draft invoice"

Selección_434

A wrong calculation of the billed lines is obtained.

Selección_435

if option "Full refund and new draft invoice" is selected, the value of the field "This credit note will be reinvoiced must always be True, and in my opinion it should be hidden.

**Comment by RodrigoBM** [comment]
> Hi @ChrisOForgeFlow ,
> 
> If you launch a wizard credit note and uncheck "This credit note will be reinvoiced" and check credit method "Full refund and new draft invoice"
> 
> Selección_434
> 
> A wrong calculation of the billed lines is obtained.
> 
> Selección_435
> 
> if option "Full refund and new draft invoice" is selected, the value of the field "This credit note will be reinvoiced must always be True, and in my opinion it should be hidden.

```
def reverse_moves(self):
    sale_qty_to_reinvoice = True if self.refund_method == "modify" else self.sale_qty_to_reinvoice
    return super(
        AccountMoveReversal,
        self.with_context(sale_qty_to_reinvoice=sale_qty_to_reinvoice),
    ).reverse_moves()
```

**Comment by RodrigoBM** [comment]
ping @ChrisOForgeFlow

**Pull request closed without merging**